### ISSUE TITLE
.pinning.store LOCK issue

### ISSUE THREAD

**Issue by simonovic86** [ceramicnetwork/js-ceramic#337]
If we run multiple Ceramic instances on the same machine, we get:

```
Error [OpenError]: IO error: lock /Users/val/Code/js-ceramic/.pinning.store/LOCK: Resource temporarily unavailable
```

The example command for running Ceramic is:

```
node packages/ceramic-cli/lib/bin/ceramic.js daemon --debug --ipfs-api=http://localhost:5102 --port=7008 --pinning-store-path=$PWD/.pinning.store.1
```

**Comment by valmack** [comment]
This should be fixed by #351

**Comment by valmack** [comment]
I think we can safely close this unless it becomes an issue again
